alcap/configure/geometry_R2015b/SiT

```diff
@@ -7,7 +7,7 @@ checkoverlap
 #==================================================================================================================
 
 DEFINE: nomThSiT 0.058 # nominal SiT thickness (dead layer needs to be included in this)
-DEFINE: thSiT  nomThSiT # -2*thDLSi-2*thDLAl # active SiT thickness
+DEFINE: thSiT  nomThSiT-2*thDLSi-2*thDLAl # active SiT thickness
```

Comment:
Oh no, this might mean I have to regenerate the MCs... how much of a difference does it make?

Reply:
I don't think it will be a lot. I went back and forth over the last few years as to whether the nominal thickness includes the dead layers of silicon and aluminium. Here I'm assuming it does but now I'm not so sure...